roles/network_plugin/calico/templates/calico-cr.yml.j2

```diff
@@ -144,6 +144,10 @@ rules:
       - ipamconfigs
     verbs:
       - get
+      - list
+      - create
+      - update
+      - delete
```

Comment:
Thanks @JaneLiuL 

From the https://github.com/projectcalico/calico/blob/master/charts/calico/templates/calico-node-rbac.yaml#L160
The List, Update, Delete is not needed.

Should it better to be removed ?

Reply:
hi, that depends on calico version, when i install kubespray with kubernetes 1.24.10 with calico version 3.24.5. we still need it.
and we should both support lower version.
I still suggest that we should keep it.

what do you think so?